resources/configurationFiles/analysisSNVCalling.xml

```diff
@@ -85,13 +87,13 @@
 
             <!--## Filter criteria -->
             <!--## The rows with corresponding values larger than below criteria will be filtered out. -->
-            <cvalue name="CRIT_ExAC_maxMAF" value="0.001" type="string" />
+            <cvalue name="CRIT_ExAC_maxMAF" value="1.0" type="string" />
```

Comment:
Whoa -- what a change? I guess this is tested, and not just a type or test setting.

Reply:
ExAC variants were included in the gnomAD WES dataset, so we just use them for annotations here, not used for the filtering.